_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -37,18 +37,15 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to the first [HTML][] `title` element that
-
-- is a [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) of the `html` element of a [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), and
-- contains [children](https://dom.spec.whatwg.org/#concept-tree-child) that are [text nodes](https://dom.spec.whatwg.org/#text) that are not only [whitespace](#whitespace).
```

Comment:
Removing the "non empty" condition does change the Applicability a bit.
I don't remember why we had this in the first place (and the split with [HTML page has non-empty title](https://act-rules.github.io/rules/2779a5) which is automatable).

Reply:
> Removing the "non empty" condition does change the Applicability a bit. I don't remember why we had this in the first place (and the split with [HTML 
 page has non-empty title](https://act-rules.github.io/rules/2779a5) which is automatable).

Thanks @Jym77 

I don't remember either :-)

But do you think we should not get rid of that condition? Does not the browser automatically discard the first title elements if it's empty?

Reply:
I'm not sure about discarding or not the condition since I do not remember why it was here in the first place (so I'm afraid there was a good reason and it would be a mistake to remove it).

I would at least want the removal to be a conscious decision in its own PR rather than a side effect hidden in an editorial PR (this is not purely editorial, and we want the changelog for the rule to list the Applicability change).

Reply:
👍 

Given the change in Applicability, I think we need to add a Failed Example with an empty `title`.